tools/build/docker/s6/services.d/redis/run

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/execlineb -P
+s6-envuidgid -B koyomi:koyomi
+/usr/bin/redis-server
```

Comment:
```suggestion
/usr/bin/redis-server /etc/redis.conf
```
I switched my home installation to a build of this container to check, and calling `/usr/bin/redis-server` directly doesn't in fact pull the configuration from `/etc` on its own!  
You have to add the path to the conf file as an argument, otherwise it'll just greet you with `No config file specified` and load its internal default.  

Which means the database isn't loaded from the database folder mountpoint, making LRR re-index everything, etc etc.